testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/settings/SettingsMenuActivity.kt

```diff
@@ -29,6 +29,10 @@ class SettingsMenuActivity : BaseActivity() {
     private lateinit var settings: AppSettings
     private lateinit var settingsProgressDialog: SettingsProgressDialog
 
+    companion object {
```

Comment:
We don't need a static define here, do we?

Reply:
`companion object` has a support to keep static fields inside. In this PR, I have used `const` as an alternative of `@JvmField`.  I could use `@JvmStatic`, but currently my intension is using it only inside this class as private.

Reply:
why do we need `companion object` here? I suppose the the normal private val declaration is enough 🤔

Reply:
Yeah, we can avoid `companion object` also. I have updated the PR.